projects/element-ng/toast-notification/si-toast.model.ts

```diff
@@ -16,6 +16,8 @@ export interface SiToast {
   timeout?: number;
   action?: Link;
   close?: () => void;
+  paused?: Subject<void>;
+  resumed?: Subject<void>;
```

Comment:
Is sthere a way to not expose those properties? I don't see any use case for this.